geoportal/c2cgeoportal_geoportal/views/entry.py

```diff
@@ -273,11 +274,9 @@ def _get_metadatas(self, item, errors):
         headers = dict(self.request.headers)
 
         # Add headers for Geoserver
-        if ogc_server.auth == main.OGCSERVER_AUTH_GEOSERVER and self.request.user is not None:
-            headers["sec-username"] = self.request.user.username
-            headers["sec-roles"] = (
-                "" if self.request.user is None else ";".join([role.name for role in self.request.user.roles])
-            )
+        if ogc_server.auth == main.OGCSERVER_AUTH_GEOSERVER:
+            headers["sec-username"] = "root"
+            headers["sec-roles"] = "root"
```

Comment:
Why do we put "root" in headers for GeoServer ?

Reply:
To do the same as for others about 10 lines higher.
To get all the layer for the internal usage in the theme building (in fact it's possible to do that without root access but it breaks the cache).